odl/set/space.py

```diff
@@ -866,20 +892,8 @@ def __ne__(self, other):
     __hash__ = None
 
     def __str__(self):
-        """Return ``str(self)``.
-
-        This is a default implementation that does not print any
-        information about the contents of the element.
-        """
-        return str(self.space) + 'Element'
-
-    def __repr__(self):
-        """Return ``repr(self)``.
-
-        This is a default implementation that does not print any
-        information about the contents of the element.
-        """
-        return repr(self.space) + 'Element'
+        """Return ``str(self)``."""
+        return repr(self)
```

Comment:
Nowhere, but the default is to just print the class name, which I guess is the most sensible thing to do here.